src/index.js

```diff
@@ -3,9 +3,16 @@ import { render } from 'react-dom'
 
 import NotesApp from 'components/NotesApp'
 
+import notesStorageService from 'state/notesStorageService'
+
 import 'styles/style.css'
 
 const startup = () => {
+  if (!notesStorageService.useIndexedDB()) {
```

Comment:
why not call this `notesStorageService.checkIndexedDB`? Essentially, nomenclature to indicate a check vs a use

Reply:
or `indexedDBAvailable`

Reply:
Because it is a *use* operation with a boolean success return value.

It does choose to use the selected service.